src/shared/components/HeaderOwnerAndContributors/HeaderOwnerAndContributors.tsx

```diff
@@ -28,15 +28,21 @@ export const HeaderOwnerAndContributors: FC<HeaderOwnerAndContributorsProps> = (
 
 	const renderContributors = (): ReactNode => {
 		if (contributors?.length) {
+			const couplingWord = tText('en');
```

Comment:
fyi, you don't need to create a variable for this neccessarily since the key that will be generated for identical translations in identical files will be the same, so only one translation entry will be made in nl.json

But in this case the variable will save some extra bytes :D